### DISCUSSION TITLE
Moving on

### DISCUSSION THREAD

**Discussion by ghost** [hedgehogqa/fsharp-hedgehog#379] [Announcements]
Hello,

Not sure if this is the "correct" place for this sort of announcement. I've been slowly transitioning to using `rust` as my primary language. Recently, I transitioned all of my personal projects away from F# and I no longer have any interest in working in the .NET ecosystem.

As a consequence of this, I will be stepping down in my role as a maintainer of Hedgehog. I wish you all the best, and I still believe you all are doing great work! It's been a pleasure working on this project.

All the best,
Adam.

/cc @moodmosaic @TysonMN @dharmaturtle

**Comment by TysonMN** [comment]
Thanks @adam-becker for your time and effort as the maintainer.  I wish you the best of luck in your journey with rust :)

**Reply by ghost** [reply]
I appreciate what we were able to accomplish in my time, thank you for the kind words.

**Comment by moodmosaic** [comment]
@adam-becker, you've done a decent amount of excellent, well thought, precise work. — Hedgehog will miss you. — Though, I'm pretty sure you can apply the same skills on the `rust` ecosystem as well, and that'll be fantastic. 🥂 

Personally, after working with C#, F#, and Haskell, for a decade (and more) I also moved on to other things. C, Node.js, and Go on the list, and I see Rust coming up as well. — I look forward collaborating together again in one of those other ecosystems.

**Reply by ghost** [reply]
I would absolutely welcome future collaboration! Can't wait for those opportunities to arise. Thanks for allowing me to take over on this project, I wish you all the best.

**Comment by dharmaturtle** [comment]
It was great working with you Adam! Professionally, I've also moved to a new language - Clojure. I'm still _very_ fond of F#, however. Until we meet again!
